app/components/windows/ChildWindow.vue.ts

```diff
@@ -57,24 +57,33 @@ export default class ChildWindow extends Vue {
     electron.remote.getCurrentWindow().setTitle(this.currentComponent.title);
   }
 
+  windowResizeTimeout: number;
+
+  windowSizeHandler() {
+    this.windowsService.updateStyleBlockers('child', true);
```

Comment:
Same comment as `Main.vue.ts` - I think we're going to be using a lot of redundant IPC doing this.